bin/uno

```diff
@@ -93,6 +93,14 @@ ashell)
   check_dirs
   "$ACCUMULO_HOME"/bin/accumulo shell -u "$ACCUMULO_USER" -p "$ACCUMULO_PASSWORD" "${@:2}"
 	;;
+astart)
+  check_dirs
```

Comment:
I could check to see if the directory ACCUMULO_HOME/bin, and similar for the others, exist.